File: Algorithms/Warmup/Compare the Triplets.cpp
/*
   Contributer : github.com/jaswal72
   Email : shubhamjaswal772@gmail.com
*/
#include <bits/stdc++.h>

using namespace std;

int main() {
    int a0,a=0,b=0;
    int a1;
    int a2;
    cin >> a0 >> a1 >> a2;
    int b0;
    int b1;
    int b2;
    cin >> b0 >> b1 >> b2;
    // Write Your Code Here
    if(a0>b0)
        ++a;
    else if(a0<b0)
        ++b;
        if(a1>b1)
        ++a;
    else if(a1<b1)
        ++b;
        if(a2>b2)
        ++a;
    else if(a2<b2)
        ++b;
        cout<<a<<" "<<b;
    return 0;
}

File: Algorithms/Warmup/Compare the Triplets.java
/*
* Contributer : github.com/shikhar-07
* Email : shikharanandg@gmail.com
*/import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int a0 = in.nextInt();
        int a1 = in.nextInt();
        int a2 = in.nextInt();
        int b0 = in.nextInt();
        int b1 = in.nextInt();
        int b2 = in.nextInt();
        int alicescore=0;
        int bobscore=0;
        if(a0>b0){
            alicescore++;
        }
        if(a1>b1){
            alicescore++;
        }
        if(a2>b2){
            alicescore++;
        }
         if(a0<b0){
            bobscore++;
        }
        if(a1<b1){
            bobscore++;
        }
        if(a2<b2){
            bobscore++;
        }
        System.out.println(alicescore+" "+bobscore);
    }
}
